content/applications/hr/lunch/orders.rst

```diff
@@ -0,0 +1,221 @@
+            beverage displayed.
+
+Your Order summary
+------------------
+
+At the top of the dashboard in the :guilabel:`Your Order` summary, the account information appears
+as well as all the information relating to all orders placed during the current calendar day.
+
+As products are added to an order, they appear at the top center of the summary box. Each product is
+listed beneath the words :guilabel:`Your Order`, with the product name, quantity, and a status tag.
+
+The available tags that can be displayed for each item are:
+
+- :guilabel:`To Order`: the product has been added to the cart, but has not been purchased yet by
+  the user.
+- :guilabel:`Ordered`: the product has been purchased by the user and is waiting to be sent to the
+  vendor by a *Lunch* app manager.
+- :guilabel:`Sent`: the order for the product has been sent to the vendor by a *Lunch* app manager.
+- :guilabel:`Received`: the product has been delivered by the vendor to the user's location, and has
+  been verified as received by a *Lunch* app manager.
+
+Product quantities can be adjusted by clicking the :guilabel:`➕ (plus sign)` or :guilabel:`➖ (minus
+sign)` to the left of the listed product. The product price adjusts in real-time to display the cost
+for the currently selected quantity of the product.
+
+The right side of the :guilabel:`Your Order` summary displays the various purchasing information.
+The :guilabel:`Total` amount for the entire day's lunch order is displayed. The :guilabel:`Already
+Paid` field indicates how much has been paid that day towards the :guilabel:`Total` amount. The
+:guilabel:`To Pay` field displays how much is required to pay to place the currently configured
+order.
+
+.. image:: orders/your-order.png
+   :align: center
+   :alt: The Your Orders section of the dashboard, with the purchasing information highlighted.
+
+.. tip::
+   Users can place multiple orders throughout the day, and are not restricted to placing only one
+   lunch order each day. Multiple orders can be placed due to forgetting to add items to an order,
+   or if there are multiple meals that are available to be purchased for the office, not just lunch.
+
+   Depending on the various vendors and how the vendors and products are configured, it is possible
+   to order breakfast, lunch, dinner, coffee, or snacks.
+
+Submit an order
+---------------
+
+To place the order, click the :guilabel:`Order Now` button on the right-side of the :guilabel:`Your
+Order` summary. The user is charged the amount that is displayed in the :guilabel:`To Pay` field,
+and the cost is deducted from their *Lunch* account balance.
+
+Once the order is placed, the tags for the items just purchased in the :guilabel:`Your Order` field
+change from orange :guilabel:`To Order` tags to red :guilabel:`Ordered` tags.
+
+Track an order
+--------------
+
+When orders have been sent to the vendors, the tags for the items in the :guilabel:`Your Order`
```

Comment:
```suggestion
When orders have been sent to the vendor(s), the tags for the items in the :guilabel:`Your Order`
```

Reply:
I believe we are not supposed to use (s) in our docs according to the new guidelines, we are suposed to just use the plural.